src/edu/wright/cs/raiderplanner/model/Room.java

```diff
@@ -27,6 +27,8 @@
  * Created by Team BRONZE on 4/27/17
  */
 public class Room extends VersionControlEntity {
+	//the value for serialVersionUID is from the value used in Activity.java
+	static final long serialVersionUID = -486727251420646703L;
```

Comment:
I have to admit that I don't have much use for serialVersionUID. However, it seems wrong to use a random-looking value that is identical to the value in another class. Can you explain why it needs to be the same as the value from Activity.java?